### PR TITLE
added Coder UH IG

### PR DESCRIPTION
**Pull request by rizkyarlin**

### CODE REVIEW

**Review by r17x — APPROVED** [review]
LGTM :rocket:

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ https://instagram.com/dsc_unhas
 ### DIMENSI STMIK DIPANEGARA
 ### XP STMIK DIPANEGARA
 ### CODER INSTITUTE UNHAS
+https://www.instagram.com/coderinstitute/
```
Comment (r17x) [inline README.md:12]: `NIT`
```diff
- https://www.instagram.com/coderinstitute/
+ ### [Coder Institute](https://www.instagram.com/coderinstitute/)
```

Comment (rizkyarlin) [reply README.md:12]: I don't think we can do this. See PR #7. Some community probably want to list multiple link. Or we just should limit to one community one URL?

Comment (r17x) [reply README.md:12]: @rizkyarlin Okay, I don't think of some community with multiple link. 

Okay God job

**Review by isfaaghyth — COMMENTED** [review]
please resolve a conflict file

**Comment by rizkyarlin** [comment]
conflict resolved

**Review by isfaaghyth — APPROVED** [review]

**Comment by isfaaghyth** [comment]
LGTM!

**Comment by arhen** [comment]
LGTM!

**Pull request merged**